From: Kamal Heib <kamalheib1@gmail.com>
Date: Thu, 19 Jul 2018 00:05:32 +0300
Subject: RDMA/mlx5: Remove set but not used variables
Patch-mainline: v4.19-rc1
Git-commit: aa09ea6e6b1279a2031cbdaa7605e2ec297e63ae
References: bsc#1103991 FATE#326007

Remove "uctx" and "pa" variables that were set but not used.

Fixes: a8b92ca1b0e5 ("IB/mlx5: Introduce DEVX")
Fixes: 8f0622873358 ("RDMA/mlx5: Remove debug prints of VMA pointers")
Signed-off-by: Kamal Heib <kamalheib1@gmail.com>
Acked-by: Leon Romanovsky <leonro@mellanox.com>
Signed-off-by: Jason Gunthorpe <jgg@mellanox.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/mlx5/devx.c |    2 --
 drivers/infiniband/hw/mlx5/main.c |    4 +---
 2 files changed, 1 insertion(+), 5 deletions(-)

--- a/drivers/infiniband/hw/mlx5/devx.c
+++ b/drivers/infiniband/hw/mlx5/devx.c
@@ -50,11 +50,9 @@ int mlx5_ib_devx_create(struct mlx5_ib_d
 	u32 in[MLX5_ST_SZ_DW(create_uctx_in)] = {0};
 	u32 out[MLX5_ST_SZ_DW(general_obj_out_cmd_hdr)] = {0};
 	u64 general_obj_types;
-	void *uctx;
 	void *hdr;
 	int err;
 
-	uctx = MLX5_ADDR_OF(create_uctx_in, in, uctx);
 	hdr = MLX5_ADDR_OF(create_uctx_in, in, hdr);
 
 	general_obj_types = MLX5_CAP_GEN_64(dev->mdev, general_obj_types);
--- a/drivers/infiniband/hw/mlx5/main.c
+++ b/drivers/infiniband/hw/mlx5/main.c
@@ -2040,7 +2040,7 @@ static int uar_mmap(struct mlx5_ib_dev *
 	struct mlx5_bfreg_info *bfregi = &context->bfregi;
 	int err;
 	unsigned long idx;
-	phys_addr_t pfn, pa;
+	phys_addr_t pfn;
 	pgprot_t prot;
 	u32 bfreg_dyn_idx = 0;
 	u32 uar_index;
@@ -2131,8 +2131,6 @@ static int uar_mmap(struct mlx5_ib_dev *
 		goto err;
 	}
 
-	pa = pfn << PAGE_SHIFT;
-
 	err = mlx5_ib_set_vma_data(vma, context);
 	if (err)
 		goto err;
